fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/controller/FinTechAuthorizationImpl.java

```diff
@@ -58,6 +58,11 @@
 
     @Override
     public ResponseEntity fromConsentOkGET(String authId, String finTechRedirectCode, UUID xRequestID, String xsrfToken) {
+        if (!authorizeService.isAuthorized()) {
+            log.warn("fromConsentOkGET failed: user is not authorized! - path is \"/v1/" + authId + "/fromConsentOk\"");
```

Comment:
Can be done without string concat:
`
log.warn("fromConsentOkGET failed: user is not authorized! - path is \"/v1/{}/fromConsentOk\"", authId);
`

Reply:
done